test/unit/lib/service_sign_in_yaml_validator_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class ServiceSignInYamlValidatorTest < ActiveSupport::TestCase
+  def service_sign_in_yaml_validator(file)
+    ServiceSignInYamlValidator.new(file)
+  end
+
+  def valid_yaml_file
+    "lib/service_sign_in/example.yaml"
+  end
+
+  def validated_hash
```

Comment:
I think I'd rename this. It is returning a valid hash, but not one validated by our validator.

Reply:
Any suggestions?

Reply:
`content`? Naming things is hard!

Reply:
Now updated 🌮